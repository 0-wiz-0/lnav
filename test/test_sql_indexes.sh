#! /bin/bash

export YES_COLOR=1

run_cap_test ${lnav_test} -n \
    -c ";EXPLAIN QUERY PLAN SELECT * FROM syslog_log WHERE log_path GLOB '*/logfile_syslog.*'" \
    ${test_dir}/logfile_syslog.*

run_cap_test ${lnav_test} -n \
    -c ";SELECT * FROM syslog_log WHERE log_path GLOB '*/logfile_syslog.*'" \
    ${test_dir}/logfile_syslog.*

