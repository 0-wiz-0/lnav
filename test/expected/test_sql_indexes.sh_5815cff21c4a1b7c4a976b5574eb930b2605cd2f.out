[1mlog_line [0m[1mlog_part [0m[1m       log_time         [0m[1m[7mlog_idle_msecs [0m[1mlog_level [0m[1mlog_mark [0m[1mlog_comment [0m[1mlog_tags [0m[1mlog_filters [0m[1mlog_hostname [0m[1mlog_msgid [0m[1mlog_pid [0m[1mlog_pri [0m[1mlog_procname [0m[1mlog_struct [0m[1m log_syslog_tag  [0m[1msyslog_version [0m
[1m       0[0m[1m [0m[1m  <[0m[1mNULL[0m[1m>[0m[1m [0m[1m2006-12-03 09:23:38.000[0m[1m [0m[1m             0[0m[1m [0m[1merror    [0m[1m [0m[1m       0[0m[1m [0m[1m     <[0m[1mNULL[0m[1m>[0m[1m [0m[1m  <[0m[1mNULL[0m[1m>[0m[1m [0m[1m     <[0m[1mNULL[0m[1m>[0m[1m [0m[1mveridian    [0m[1m [0m[1m   <[0m[1mNULL[0m[1m>[0m[1m [0m[1m7998   [0m[1m [0m[1m <[0m[1mNULL[0m[1m>[0m[1m [0m[1mautomount   [0m[1m [0m[1m    <[0m[1mNULL[0m[1m>[0m[1m [0m[1mautomount[7998] [0m[1m [0m[1m        <[0m[1mNULL[0m[1m> [0m
       1   <NULL> 2006-12-03 09:23:38.000              0 info             0      <NULL>   <NULL>      <NULL> veridian        <NULL> 16442    <NULL> automount        <NULL> automount[16442]         <NULL> 
[1m       2[0m[1m [0m[1m  <[0m[1mNULL[0m[1m>[0m[1m [0m[1m2006-12-03 09:23:38.000[0m[1m [0m[1m             0[0m[1m [0m[1merror    [0m[1m [0m[1m       0[0m[1m [0m[1m     <[0m[1mNULL[0m[1m>[0m[1m [0m[1m  <[0m[1mNULL[0m[1m>[0m[1m [0m[1m     <[0m[1mNULL[0m[1m>[0m[1m [0m[1mveridian    [0m[1m [0m[1m   <[0m[1mNULL[0m[1m>[0m[1m [0m[1m7999   [0m[1m [0m[1m <[0m[1mNULL[0m[1m>[0m[1m [0m[1mautomount   [0m[1m [0m[1m    <[0m[1mNULL[0m[1m>[0m[1m [0m[1mautomount[7999] [0m[1m [0m[1m        <[0m[1mNULL[0m[1m> [0m
[7m [0m      3   <NULL> 2007-01-03 09:47:02.000     2679804000 info             0      <NULL>   <NULL>      <NULL> veridian        <NULL> <NULL>   <NULL> sudo             <NULL> sudo                     <NULL> 
[1m[7m       4[0m[1m[7m [0m[1m[7m  <[0m[1mNULL[0m[1m>[0m[1m [0m[1m2007-11-03 09:23:38.000[0m[1m [0m[1m   26264196000[0m[1m [0m[1merror    [0m[1m [0m[1m       0[0m[1m [0m[1m     <[0m[1mNULL[0m[1m>[0m[1m [0m[1m  <[0m[1mNULL[0m[1m>[0m[1m [0m[1m     <[0m[1mNULL[0m[1m>[0m[1m [0m[1mveridian    [0m[1m [0m[1m   <[0m[1mNULL[0m[1m>[0m[1m [0m[1m7998   [0m[1m [0m[1m <[0m[1mNULL[0m[1m>[0m[1m [0m[1mautomount   [0m[1m [0m[1m    <[0m[1mNULL[0m[1m>[0m[1m [0m[1mautomount[7998] [0m[1m [0m[1m        <[0m[1mNULL[0m[1m> [0m
       5   <NULL> 2007-11-03 09:23:38.000              0 info             0      <NULL>   <NULL>      <NULL> veridian        <NULL> 16442    <NULL> automount        <NULL> automount[16442]         <NULL> 
[1m       6[0m[1m [0m[1m  <[0m[1mNULL[0m[1m>[0m[1m [0m[1m2007-11-03 09:23:38.000[0m[1m [0m[1m             0[0m[1m [0m[1merror    [0m[1m [0m[1m       0[0m[1m [0m[1m     <[0m[1mNULL[0m[1m>[0m[1m [0m[1m  <[0m[1mNULL[0m[1m>[0m[1m [0m[1m     <[0m[1mNULL[0m[1m>[0m[1m [0m[1mveridian    [0m[1m [0m[1m   <[0m[1mNULL[0m[1m>[0m[1m [0m[1m7999   [0m[1m [0m[1m <[0m[1mNULL[0m[1m>[0m[1m [0m[1mautomount   [0m[1m [0m[1m    <[0m[1mNULL[0m[1m>[0m[1m [0m[1mautomount[7999] [0m[1m [0m[1m        <[0m[1mNULL[0m[1m> [0m
[7m [0m      7   <NULL> 2007-11-03 09:47:02.000        1404000 info             0      <NULL>   <NULL>      <NULL> veridian        <NULL> <NULL>   <NULL> sudo             <NULL> sudo                     <NULL> 
[1m[7m       8[0m[1m[7m [0m[1m[7m  <[0m[1m[7mNULL[0m[1m[7m>[0m[1m[7m [0m[1m[7m2021-11-03 09:23:38.000[0m[1m[7m [0m[1m[7m  441848196000[0m[1m[7m [0m[1m[7minfo     [0m[1m[7m [0m[1m[7m       0[0m[1m[7m [0m[1m[7m     <[0m[1m[7mNULL[0m[1m[7m>[0m[1m[7m [0m[1m[7m  <[0m[1m[7mNULL[0m[1m[7m>[0m[1m[7m [0m[1m[7m     <[0m[1m[7mNULL[0m[1m[7m>[0m[1m[7m [0m[1m[7mveridian    [0m[1m[7m [0m[1m[7m   <[0m[1m[7mNULL[0m[1m[7m>[0m[1m[7m [0m[1m[7m7998   [0m[1m[7m [0m[1m[7m <[0m[1m[7mNULL[0m[1m[7m>[0m[1m[7m [0m[1m[7mfoo         [0m[1m[7m [0m[1m[7m    <[0m[1m[7mNULL[0m[1m[7m>[0m[1m[7m [0m[1m[7mfoo[7998]       [0m[1m[7m [0m[1m[7m        <[0m[1m[7mNULL[0m[1m[7m>[0m[1m [0m
       9   <NULL> 2021-11-03 09:23:38.000              0 info             0      <NULL>   <NULL>      <NULL> veridian        <NULL> 16442    <NULL> foo              <NULL> foo[16442]               <NULL> 
[1m      10[0m[1m [0m[1m  <[0m[1mNULL[0m[1m>[0m[1m [0m[1m2021-11-03 09:23:38.000[0m[1m [0m[1m             0[0m[1m [0m[1minfo     [0m[1m [0m[1m       0[0m[1m [0m[1m     <[0m[1mNULL[0m[1m>[0m[1m [0m[1m  <[0m[1mNULL[0m[1m>[0m[1m [0m[1m     <[0m[1mNULL[0m[1m>[0m[1m [0m[1mveridian    [0m[1m [0m[1m   <[0m[1mNULL[0m[1m>[0m[1m [0m[1m7999   [0m[1m [0m[1m <[0m[1mNULL[0m[1m>[0m[1m [0m[1mfoo         [0m[1m [0m[1m    <[0m[1mNULL[0m[1m>[0m[1m [0m[1mfoo[7999]       [0m[1m [0m[1m        <[0m[1mNULL[0m[1m> [0m
